src/Azure/AzureAD/Authentication.AzureAD.UI/test/AzureADAuthenticationBuilderExtensionsTests.cs

```diff
@@ -76,6 +76,51 @@ public void AddAzureAD_ConfiguresAllOptions()
             Assert.Equal(AzureADDefaults.CookieScheme, openIdOptions.SignInScheme);
         }
 
+        [Fact]
+        public void AddAzureAD_AllowsOverridingCookiesAndOpenIdConnectSettings()
```

Comment:
Can you add a variant that also has an existing bearer/cookie for these tests, i.e.

`services.AddAuthentication().AddCookie().AddAzureAD()`
`services.AddAuthentication().AddBearer().AddAzureADBearer()`

since that was one of the broken scenarios before your fix.

Reply:
Done